packages/quantic/force-app/main/default/lwc/quanticCaseClassification/__tests__/quanticCaseClassification.test.js

```diff
@@ -0,0 +1,848 @@
+              exampleCaseClassificationSuggestions[
+                automaticallySelectedSuggestionIndex
+              ].value,
+              {caseClassifications: false, documentSuggestions: false},
+              true
+            );
+
+            const suggestionInputs = element.shadowRoot.querySelectorAll(
+              selectors.caseClassificationSuggestionInput
+            );
+
+            suggestionInputs[exampleSelectedSuggestionIndex].click();
+            await flushPromises();
+
+            expect(functionsMocks.update).toHaveBeenCalledTimes(2);
+            expect(functionsMocks.update).toHaveBeenCalledWith(
+              exampleCaseClassificationSuggestions[
+                exampleSelectedSuggestionIndex
+              ].value,
+              {caseClassifications: false, documentSuggestions: false},
+              undefined
+            );
+          });
+        });
+      });
+
+      describe('the case classification all options', () => {
+        it('should display the show select input button', async () => {
+          const element = createTestComponent({
+            ...defaultOptions,
+            maxSuggestions: exampleMaxSuggestions,
+          });
+          mockGetPicklistValuesByRecordTypeWire();
+
+          await flushPromises();
+
+          const showSelectInputButton = element.shadowRoot.querySelector(
+            selectors.showSelectInputButton
+          );
+
+          expect(showSelectInputButton).not.toBeNull();
+          expect(showSelectInputButton.title).toBe(exampleMoreTopicsLabel);
+          expect(showSelectInputButton.label).toBe(exampleMoreTopicsLabel);
+        });
+
+        describe('when clicking the show select input button', () => {
+          it('should display the all options select input', async () => {
+            const element = createTestComponent({
+              ...defaultOptions,
+              maxSuggestions: exampleMaxSuggestions,
+            });
+            mockGetPicklistValuesByRecordTypeWire();
+
+            await flushPromises();
+
+            const showSelectInputButton = element.shadowRoot.querySelector(
+              selectors.showSelectInputButton
+            );
+            showSelectInputButton.click();
+            await flushPromises();
+
+            const allOptionsSelectInput = element.shadowRoot.querySelector(
+              selectors.allOptionsSelectInput
+            );
+
+            expect(allOptionsSelectInput).not.toBeNull();
+            expect(allOptionsSelectInput.options).toStrictEqual(
+              exampleCaseClassificationAllOptions
+            );
+            expect(allOptionsSelectInput.value).toBe(
+              exampleCaseClassificationSuggestions[0].value
+            );
+            expect(allOptionsSelectInput.placeholder).toBe(
+              exampleMoreTopicsLabel
+            );
+          });
+
+          describe('when selecting an option from the all options select input', () => {
```

Comment:
Not sure you need this level of describe :D

Reply:
true 👍